server/knexfile.js

```diff
@@ -17,31 +17,9 @@ module.exports = {
     }
   },
 
-  staging: {
-    client: "postgresql",
-    connection: {
-      host: process.env.DB_HOST,
-      database: process.env.DB_NAME,
-      user: process.env.DB_USER,
-      password: process.env.DB_PASSWORD
-    },
-    pool: {
-      min: 2,
-      max: 10
-    },
-    migrations: {
-      tableName: "knex_migrations"
-    }
-  },
-
   production: {
-    client: "postgresql",
-    connection: {
```

Comment:
how come this works and it still connects to the DB without user/password?

Reply:
The host/user/password is encoded into the connection string.